test/fixtures/deploys.yml

```diff
@@ -15,3 +15,21 @@ succeeded_production_test:
   started_at: 2014-01-02 20:00:00
   updated_at: 2014-01-02 20:10:00
   created_at: 2014-01-02 19:50:00
+
+succeeded_master_test:
```

Comment:
modify existing fixtures to suit your usecase, we pay a price on every test run for every fixture we add